packages/scripts/package.json

```diff
@@ -60,7 +60,7 @@
 		"minimist": "^1.2.0",
 		"npm-package-json-lint": "^5.0.0",
 		"prettier": "npm:wp-prettier@1.19.1",
-		"puppeteer": "^2.0.0",
+		"puppeteer": "npm:puppeteer-core@3.0.0",
```

Comment:
It looks like aliasing is required only temporary until new version of `jest-puppeteer-environments` is published:
https://github.com/smooth-code/jest-puppeteer/pull/315

Reply:
The e2e tests are failing in https://github.com/WordPress/wordpress-develop/pull/347. Does this have something to do with it?